internal/faustjs.org/docs/apollo.mdx

```diff
@@ -39,10 +39,57 @@ It co-locates the GraphQL query within the same file. You can also import and us
 The benefit with Faust is that you don't have to create the client object in your codebase. It is automatically created when you first import the
 `@faustwp-core` package. Instead you can customize it by using a [plugin filter](https://faustjs.org/docs/plugin-system/filters).
 
+## Using Persisted Queries
+
+Faust has built in support for persisted queries with Apollo Client as of 0.3.0. This is helpful for tools like [WPGraphQL Smart Cache](https://wordpress.org/plugins/wpgraphql-smart-cache/), which caches your WPGraphQL requests and automatically evicts those cached requests when the data changes.
+
+To enable persisted queries in your Faust app, you can add the following property to your `faust.config.js`:
+
+```jsx title="faust.config.js" {12}
+import { setConfig } from '@faustwp/core';
+import templates from './wp-templates';
+import possibleTypes from './possibleTypes.json';
+
+/**
+ * @type {import('@faustwp/core').FaustConfig}
+ **/
+export default setConfig({
+  templates,
+  experimentalPlugins: [],
+  possibleTypes,
+  usePersistedQueries: true,
+});
+```
+
+## Changing the Desired HTTP Method for Your GraphQL Requests
+
+Since Faust 0.3.0, GraphQL requests are made using the `GET` HTTP method. Since most hosting providers cache `GET` requests, this will help keep your WordPress instance below capacity. However, there are some instances where you may want Apollo to use the `POST` HTTP method for GraphQL requests:
+
+- You are receiving stale data
+- Your GraphQL queries are too long to be stringified into a URL (this length is determined by your hosting provider)
+
+If you want Faust to use `POST` requests instead of `GET` requests, you can set the following property in your `faust.config.js`:
```

Comment:
Love this. Would it be helpful to also show how to make one-off POST requests, while leaving all others GET?

Reply:
Love this idea. Just added it @mindctrl

Reply:
Nice!